Expose PutHumptyTogetherAgain method in LfMergeBridge

rmunn opened the pull request:
The LexBox project wants to be able to [generate .fwdata files on the fly](https://github.com/sillsdev/languageforge-lexbox/issues/631), which will enable various features such as being able to view recent changes to a project in a nicer form than Mercurial diffs. In order to make this work, we need to expose the PutHumptyTogetherAgain method as a public method.

This PR seems the simplest way to do it, enabling access to that API from LfMergeBridge without changing the access granted to any other projects. That way we minimize the risk to FLEx and WeSay, as the code they use is not touched at all.

<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/sillsdev/flexbridge/398)
<!-- Reviewable:end -->

rmunn committed:
Expose PutHumptyTogetherAgain method

Will be used for LexBox to be able to generate .fwdata files as needed

jasonleenaylor reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/398)* status: 0 of 1 files reviewed, all discussions resolved

___
*[`src/LfMergeBridge/LfMergeBridge.cs` line 75 at r1](https://reviewable.io/reviews/sillsdev/flexbridge/398#-NvmG8n-DXQkciwC4CAm:-NvmG8n-DXQkciwC4CAn:b-tkaakj) ([raw file](https://github.com/sillsdev/flexbridge/blob/77899f1d73cc58306bb06037233bea387d9b1ca6/src/LfMergeBridge/LfMergeBridge.cs#L75)):*
> ```Smalltalk
> 		}
> 
> 		public static void PutHumptyTogetherAgain(IProgress progress, bool writeVerbose, string mainFilePathname)
> ```

This is an opportunity, should you want one, to adjust the name.



_Code quote:_
```Smalltalk
PutHumptyTogetherAgain
```


<!-- Sent from Reviewable.io -->

rmunn committed:
Switch name of fwdata file API

The Humpty name is fun and cleverly silly, but when we expose it to
outside code in LfMerge, it might be nicer to have a name that
communicates its purpose a little more clearly.

rmunn reviewed (commented):
> *[Reviewable](https://reviewable.io/reviews/sillsdev/flexbridge/398)* status: 0 of 1 files reviewed, all discussions resolved (waiting on @jasonleenaylor)

___
*[`src/LfMergeBridge/LfMergeBridge.cs` line 75 at r1](https://reviewable.io/reviews/sillsdev/flexbridge/398#-NvmG8n-DXQkciwC4CAm:-NwD6149AnTslQp9v-DX:b5xfba0) ([raw file](https://github.com/sillsdev/flexbridge/blob/77899f1d73cc58306bb06037233bea387d9b1ca6/src/LfMergeBridge/LfMergeBridge.cs#L75)):*
<details><summary><i>Previously, jasonleenaylor (Jason Naylor) wrote…</i></summary><blockquote>

This is an opportunity, should you want one, to adjust the name.
</blockquote></details>

Good point. I've pushed a new commit to use more boring, but easier to understand, names. And while I'm at it, I've decided to expose the reverse operation too; I have no need for it *yet* in the code I'm about to write in LexBox, but could definitely see a need for this in future features.


<!-- Sent from Reviewable.io -->

rmunn commented:
I'm assuming Jason would have mentioned any objections if he had them; since this affects nothing except the code that interfaces with LfMerge, I'm going to go ahead and merge it now.
